images/dpu-intel-ipu-p4sdk.yml

```diff
@@ -0,0 +1,32 @@
+arches:
+- aarch64
+content:
+  source:
+    dockerfile: Dockerfile.IntelP4.rhel
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/dpu-operator.git
+      web: https://github.com/openshift/dpu-operator
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          member: ci-openshift-build-root-latest.rhel9
+dependents:
+- dpu-operator
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: dpu-intel-ipu-p4sdk-container
+delivery:
+  repo_name: ose-dpu-intel-ipu-p4sdk
+for_payload: false
+from:
+  member: openshift-enterprise-base-rhel9
+name: openshift/ose-dpu-intel-ipu-p4sdk-rhel9
```

Comment:
Image build does not work yet in Konflux, could you disable that build system here?